content/spin/v3/cron-trigger.md

```diff
@@ -0,0 +1,146 @@
+title = "The Spin Cron Trigger"
+template = "spin_main"
+date = "2025-01-09T00:00:01Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/v3/cron-trigger.md"
+
+---
+
+- [Cron Trigger Expressions](#cron-trigger-expressions)
+- [Installing the Cron Trigger](#installing-the-cron-trigger)
+  - [Installing the Cron Trigger Plugin](#installing-the-cron-trigger-plugin)
+  - [Installing the Cron Trigger Template](#installing-the-cron-trigger-template)
+- [Creating the Application](#creating-the-application)
+  - [Inspecting the Source Code](#inspecting-the-source-code)
+- [Building and Running the Application](#building-and-running-the-application)
+- [Additional Spin Triggers](#additional-spin-triggers)
+  - [Common Triggers](#common-triggers)
+  - [Community Triggers](#community-triggers)
+
+
+Spin has experimental support for creating and running components on a schedule. Please note that there are only working Cron Trigger app samples written in [Rust](https://github.com/fermyon/spin-trigger-cron/tree/main/guest-rust) and [Python](https://github.com/fermyon/spin-trigger-cron/tree/main/guest-python) at present.
+
+> Please note: You can not `spin deploy` an application to Fermyon Cloud if it uses `cron` because non-HTTP triggers are not supported in Fermyon Cloud.
+
+Let's look at how the [experimental Cron trigger for Spin](https://github.com/fermyon/spin-trigger-cron) allows you to deploy an application that runs on a schedule. A Cron trigger maps a cron expression (a schedule) to a specific component. For example:
+
+<!-- @nocpy -->
+
+```toml
+[[trigger.cron]]
+component = "hello-cron"
+cron_expression = "1/2 * * * * *"
+```
+
+> Note: The 7th field (year) for the `cron_expression` is optional.
+
+## Cron Trigger Expressions
+
+The expression is based on the crontab (cron table) syntax whereby each line is made up of 7 fields that represent the time to execute.
+
+```bash
+#  ┌──────────── sec (0–59)
+#  |    ┌───────────── min (0–59)
+#  |    │  ┌───────────── hour (0–23)
+#  |    │  │  ┌───────────── day of month (1–31)
+#  |    │  │  │  ┌───────────── month (1–12)
+#  |    │  │  │  │  ┌───────────── day of week (0–6)
+#  |    │  │  │  │  |  ┌─────────────- year
+#  |    │  │  │  │  |  │
+#  |    │  │  │  │  |  │
+  1/30  *  *  *  *  *  * 
+```
+
+> For more information about setting the schedule, please see the [Spin Cron Trigger repository](https://github.com/fermyon/spin-trigger-cron?tab=readme-ov-file#trigger-configuration). 
+
+Let's look at a time-based workload inside a Rust application.
+
+## Installing the Cron Trigger
+
+### Installing the Cron Trigger Plugin
+
+First, we install the plugin:
+
+```bash
+spin plugins install --url https://github.com/fermyon/spin-trigger-cron/releases/download/canary/trigger-cron.json
```

Comment:
Again, would prefer not to point to a canary, as these are a bit "wild west."  We should create a release so that we know what we are pointing people to, even if we don't yet feel ready to put it in the registry.  I've raised an issue on the cron trigger repo for this.